lib/algos/list/delimiterMaster.js

```diff
@@ -116,7 +117,7 @@ class DelimiterMaster extends Delimiter {
     }
 
     skipping() {
-        if (this.NextMarker) {
+        if (this.NextMarker && this.NextMarker !== this.markerParams) {
```

Comment:
Does this mean that if we haven't updated the NextMarker, `skipping()` will return `SKIP_NONE` ?
I don't see how that improves the situation. Was there a new corner case identified since the last version of the fix ?